src/client/src/rt-components/platform/adapters/glue/window.ts

```diff
@@ -0,0 +1,191 @@
+import { WindowConfig } from '../types'
+import { Glue42 } from 'tick42-glue'
+
+type BrowserWindowProps = WindowConfig
+type GDWindow = Glue42.Windows.GDWindow
+
+let listOfOpenedWindows: GDWindow[] = []
+
+/**
+ * Glue42 has the ability to manage windows.The registered methods here give the ability to demonstrate different
+ * functionalities with the opened windows (Activity, Blotter, Spot windows)
+ */
+export const registerWindowMethods = (glue: any) => {
+  if (location.href.indexOf('spot') >= 0) {
```

Comment:
it is possible that we will change the route names in future, causing this to fail.

Reply:
What would be a reliable way to check if the window is a spot?